test/wdio.conf.sauce.test.js

```diff
@@ -1,26 +1,19 @@
-exports.config = {
-  specs: ['./test/e2e/**/*.js'],
-  exclude: [],
-  maxInstances: 1,
+const merge = require('deepmerge')
+const wdioConf = require('./wdio.conf')
+
+exports.config = merge(wdioConf.config, {
   services: ['sauce'],
   user: process.env.SAUCE_USERNAME,
   key: process.env.SAUCE_ACCESS_KEY,
   baseUrl: process.env.EXT_WEB_TEST_BASEURL || 'http://localhost:3000',
   capabilities: [{
-    maxInstances: 1,
-    browserName: 'firefox',
-    platform: 'Windows 10',
-    version: '49.0'
+    platformName: 'iOS',
+    platformVersion: '9.1',
+    browserName: 'Safari',
+    deviceName: 'iPhone 6s Simulator',
+    deviceOrientation: 'portrait'
   }],
-  sync: false,
-  logLevel: 'verbose',
-  coloredLogs: true,
-  screenshotPath: './errorShots/',
-  waitforTimeout: 10000,
-  connectionRetryTimeout: 90000,
-  connectionRetryCount: 3,
-  framework: 'mocha',
-  mochaOpts: {
-    ui: 'bdd'
-  }
-}
+  browserName: 'firefox',
+  platform: 'Windows 10',
+  version: '49.0'
+})
```

Comment:
I think this is a copy/paste error on my part.  I'm gonna revert the values back to what they were before, and remove any of the values that are being set in the base config file